javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,152 @@
+    let inMinutes = (arivalTime - Math.floor(arivalTime)) * 60;
+    return `${Math.floor(arivalTime)} hours and ${Math.floor(inMinutes)} minutes`;
+}
+  
+  
+  const travelTime = whenWillBeThere(travelInformation);
+  console.log(travelTime); // 8 hours and 38 minutes
+
+  //Series duration of my life
+  const seriesDurations = [
+    {
+      title: "Friends",
+      days: 10,
+      hours: 4,
+      minutes: 20,
+    },
+    {
+      title: "The Big Bang Theory",
+      days: 14,
+      hours: 10,
+      minutes: 38,
+    },
+    {
+      title: "Breaking bad",
+      days: 14,
+      hours: 10,
+      minutes: 38,
+    },
+  ];
+ 
+  
+  function logOutSeriesText() {
+    const age = 80;
+    const ageInHours = age * 365 * 24;
+    let amount = 0;
+
+    for (let i = 0; i < seriesDurations.length; i++) {
+      let hoursTime = seriesDurations[i].days * 24 + seriesDurations[i].hours;
+      let persentOfLife = hoursTime * 100 / ageInHours;
+      console.log(`${seriesDurations[i].title} took ${persentOfLife.toFixed(3)} % of my life`);
+      }
+    }
+  
+  logOutSeriesText(); // logs out the text found above
+
+
+  //NOnoN0nOYes (Note taking app)
+
+  const notes = [];
+
+function saveNote(content, id) {
+  notes.push({"content": content, "id": id})// write some code here
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+
+//_____Get a note___________
+
+function getNote(id) {
+  for (let i = 0; i < notes.length; i++){
+    if (notes[1].id === id || id !== Number){
+      return notes[i];
+    } else return "Error"; 
+    } }
+ 
+const firstNote = getNote(1);
+console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+
+//______Log out notes_______
+
+function logOutNotesFormatted() {
+  for (let i = 0; i < notes.length; i++){
+    return console.log(`The note with id: ${notes[i].id}, has the following note text: ${notes[i].content}`)// your code here
+}
+}
+logOutNotesFormatted(); 
+
+//CactusIO-interactive (Smart phone usage app) optional
+
+const activities = [];
+const date = new Date().toLocaleString();
+
+function addActivity(date, activity, duration){
+if (typeof activity === "string" && typeof duration === "number"){
+  activities.push({"date": date, "activity": activity, "duration": duration});
+  return activities;
+} else {
+  return "Error!!!"
+}
+}
+
+addActivity("23/7-18", "Youtube", 30);
+/*
+activities should now look like this
+[{
+    date: '23/7-18',
+    activity: 'Youtube',
+    duration: 30,
+}]
+*/
+
+function showStatus() {
+  let allTime = 0;
+
+  for (let i = 0; i < activities.length; i++){
+    allTime += activities[i].duration;  
+    if (activities.length === 0) {
+    console.log("Add some activities before calling showStatus")
+  } else {
+    return `You have added ${activities[i].activity} activities. They amount to ${allTime} min. of usage`
+
```

Comment:
remember 'return' functions like a break. if you have more than one activities, the showStatus will only output the first one.